protocol-doc.md

```diff
@@ -203,6 +203,14 @@ represented by a space), and the rest of the string being their username.
 
 > Indicates that the game has started.
 
+`|win|USER`
+
+> `USER` has won the battle.
+
+`|tie`
```

Comment:
I was under the impression that ties were now impossible? Or is this just a reverse compatability thing?